feature-configs/cn-ran-overlays/all-in-one/cluster/deployment/performance/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
```

Comment:
I think all the deployment stuff can be based on 
https://github.com/openshift-kni/cnf-features-deploy/tree/master/feature-configs/deploy
ie for PAO
add 
```
bases:
  - ../../../../deploy/performance
```

then just kustomize whatever you need (which I think is nothing ATM)

Reply:
Yep, same thing I proposed in https://github.com/openshift-kni/cnf-features-deploy/pull/377#discussion_r532491759

Reply:
I asked them to decouple this though as there is nothing to gain from the inheritance here. They have a different hardware and a different profile.

Reply:
Oh, scratch this, I see it is the same repo..